### DISCUSSION TITLE
Allow Third Party Cookies on newer versions of chromedriver

### DISCUSSION THREAD

**Discussion by NazarOliynyk** [webdriverio/webdriverio#12381] [Community]
Hi, 

I encountered the problem with chromedriver 121 and above - it looks like _**Google Chrome 121** introduced changes to its handling of **third-party cookies**, which could potentially affect how automated tests interact with websites that rely on third-party cookies for various functionalities._

approaches such as **args** :
_--allow-third-party-cookies,
--disable-web-security,
--disable-site-isolation-trials,
--disable-incognito,
--no-sandbox_,

    or **prefs** : 
_'profile.default_content_setting_values.cookies': 1,
'profile.block_third_party_cookies': false,
'profile.content_settings.exceptions.cookies.*.setting': 1_

    **are not working** - it seems that chrome under test just overrides such security work-arounds.

Is there any way to **forcefully allow** third-party-cookies on the chrome under automated tests ?

**Comment by christian-bromann** [comment]
@NazarOliynyk I would recommend to post Chrome specific questions in the Puppeteer project as the team that is working on that project is most knowledgable about this things.

**Reply by NazarOliynyk** [reply]
OK - I will raise this question there as well.

Could You please also suggest the link to Puppeteer project ?

**Reply by christian-bromann** [reply]
https://pptr.dev/

**Reply by NazarOliynyk** [reply]
ok- thank you very much

**Reply by ronaldiansana** [reply]
Any updates about your issue sir?

**Reply by vitalii-shemetov** [reply]
Can you suggest any workaround ? It blocks my automation testing. Maybe I should downgrade Chrome/driver version ?

**Comment by ronaldiansana** [comment]
Any updates about the issue?

**Reply by christian-bromann** [reply]
@ronaldiansana this is a Chrome related issue and you should seek help in the Chrome project not WebdriverIO.
